app/views/storage_locations/show.html.erb

```diff
@@ -110,9 +110,6 @@
                       </tr>
                     <% end %>
                   <% else %>
-                    <%= render partial: "inventory_item_row",
-                               collection: @storage_location.inventory_items.joins(:item).where(items: { active: true }),
-                               locals: { version_date: params[:version_date] } %>
                   <% end %>
```

Comment:
No longer need the else at all, and maybe don't need to `elsif @legacy_inventory` block either?

Reply:
Heh I fell into that trap. We *do* need this because it tracks inventory from before event sourcing was introduced.